actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.sunbird.feed;
+
+import java.util.*;
+import java.util.stream.IntStream;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.bean.ShadowUser;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.feed.impl.FeedFactory;
+import org.sunbird.models.user.Feed;
+
+/** this class will be used as a Util for inserting Feed in table */
+public class FeedUtil {
+  private static IFeedService feedService = FeedFactory.getInstance();
+  private static final String ORG_MIGRATION_ACTION = "OrgMigrationAction";
```

Comment:
better to create feedAction enum , so that later we can easily add new feedtype.